includes.mk

```diff
@@ -0,0 +1,12 @@
+COMPONENT ?= workflow
```

Comment:
I'm not sure this makes sense in includes.mk, since COMPONENT typically in this case should be set to `deis` for the client